charts/enterprise-kyverno-operator/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: nirmata-kyverno-operator
 description: Helm Chart for Enterprise Kyverno Operator
 type: application
-version: v0.4.19-rc1
-appVersion: v0.3.15-rc1
+version: v0.4.19-rc2
```

Comment:
@VedRatan already the v0.4.19 is the GA version of kyverno operator. We need to create v0.4.20-rc.1 i believe

Reply:
done, @nsathyaseelan thanks for pointing this out.